core/src/main/java/hudson/security/HudsonPrivateSecurityRealm.java

```diff
@@ -839,6 +839,10 @@ public Details newInstance(StaplerRequest2 req, JSONObject formData) throws Form
                     throw new FormException("Please confirm the password by typing it twice", "user.password2");
                 }
 
+                if (FIPS140.useCompliantAlgorithms() && pwd.length() < FIPS_PASSWORD_LENGTH) {
+                     throw new FormException(Messages.HudsonPrivateSecurityRealm_CreateAccount_FIPS_PasswordLengthInvalid(), "user.password1");
```

Comment:
Isn't the form field here `user.password` instead of `user.password1` as per line 834?

Reply:
That's right , Thank you for pointing out!!